api/server/structs/conversions_lightclient.go

```diff
@@ -20,29 +20,39 @@ func LightClientUpdateFromConsensus(update interfaces.LightClientUpdate) (*Light
 	if err != nil {
 		return nil, errors.Wrap(err, "could not marshal finalized light client header")
 	}
-	finalityBranch := update.FinalityBranch()
 
 	var scBranch [][32]byte
+	var finalityBranch [][32]byte
 	if update.Version() >= version.Electra {
-		b, err := update.NextSyncCommitteeBranchElectra()
+		scb, err := update.NextSyncCommitteeBranchElectra()
 		if err != nil {
 			return nil, err
 		}
-		scBranch = b[:]
+		scBranch = scb[:]
+		fb, err := update.FinalityBranchElectra()
+		if err != nil {
+			return nil, err
+		}
+		finalityBranch = fb[:]
 	} else {
 		b, err := update.NextSyncCommitteeBranch()
```

Comment:
```suggestion
		scb, err := update.NextSyncCommitteeBranch()
```
Lets have this for consistency